pkg/workflow/common/const.go

```diff
@@ -63,8 +63,8 @@ const (
 
 	// WorkflowLabelName is label to indicate resources created by Cyclone workflow engine
 	WorkflowLabelName = "cyclone.io/workflow"
-	// WorkflowRunLabelName is label applied to WorkflowRun to specify Workflow
-	WorkflowRunLabelName = "cyclone.io/workflow-name"
+	// WorkflowNameLabelName is label applied to WorkflowRun to specify Workflow
+	WorkflowNameLabelName = "cyclone.io/workflow-name"
```

Comment:
I'd suggest to extract these label const to `pkg/common`, so that cyclone server and workflow engine can share them. You can create a new PR for it.